webapp/loggedin/modules/users/user/components/profilePictureEditor.scss

```diff
@@ -0,0 +1,56 @@
+@import "../../../../../style/vars";
+
+.profile-picture-editor {    
+  height: 260px;
+  display: flex;
+  justify-content: center;
+}
+
+.profile-picture-editor__sliders {
+  justify-content: center;
+  grid-row-gap: 0.5rem;
+
+  .slider {
+    padding: 0;
+    -webkit-appearance: none;
+    height: 8px;
+    border-radius: 4px;   
+    background: $blueLight;
+    outline: none;
+    opacity: 0.7;
+
+    &:focus {
+      background-color: $blueLight;
+    }
+
+    &:disabled {
+      background: rgba($blueLight, .5);
+      &::-webkit-slider-thumb {
+        background: rgba($blue, .3);
+      }
+      &::-moz-range-thumb {
+        background: rgba($blue, .3);
```

Comment:
it's the same as above, please combine them with a comma

Reply:
Right